lib/grape.rb

```diff
@@ -133,6 +134,8 @@ module Presenters
   end
 end
 
+include Grape::Http::Headers
```

Comment:
Maybe group with the above `require`, but in theory this shouldn't even be necessary since this class should be found anyway, no?